### PR TITLE
Bpk 1109 rethink bpk image positoning

### PR DESCRIPTION
**Pull request by k0nserv**

### CODE REVIEW

File: packages/bpk-component-image/src/withLazyLoading.js [hunk 1]
```diff
@@ -103,19 +104,32 @@ export default function withLazyLoading(Component, document) {
...
             inView={this.state.inView}
-            {...this.props}
+            {...rest}
           />
         </div>
       );
     }
   }
   WithLazyLoading.displayName = wrapDisplayName(Component, 'withLazyLoading');
+  WithLazyLoading.propTypes = {
```
Comment (matthewdavidson) [inline packages/bpk-component-image/src/withLazyLoading.js]: My eyes 😭  (new lines)

File: changelog.md [hunk 2]
```diff
@@ -6,11 +6,17 @@
 - bpk-scrim-utils:
   - Removed `padded` prop in favour of more generic `contentClassName`
   - Fixed modal white background on iPhone not showing
+- bpk-component-image:
+  - `withLazyLoading` now applys styling options to the wrapper div it creates rather than the lazy loaded component.
```
Comment (matthewdavidson) [inline changelog.md]: *applies

Comment (k0nserv) [reply changelog.md]: That's what I get for not reading what I write

**Comment by georgegillams** [comment]
When I apply a className with `margin: 50px` it causes some bizarre behaviour:
screen shot 2017-11-20 at 14 39 24

And for lazy loaded ones: 
screen shot 2017-11-20 at 14 40 25

**Comment by georgegillams** [comment]
I think the first is due to the paddingBottom: 66.66% thing, as the image is now not using the full width of the container?

**Comment by k0nserv** [comment]
@georgegillams You applied it to the aspect ratio container right? Consumers can't/shouldn't change the style on that one. All though it seems like we add any className to that div. https://github.com/Skyscanner/backpack/blob/8ebc56deba530024a7df54e526c96709ac26ba89/packages/bpk-component-image/src/BpkImage.js#L104

**Comment by k0nserv** [comment]
We should move that to be added to the outer div

**Comment by k0nserv** [comment]
See 1cad452

**Comment by georgegillams** [comment]
Yeah I applied it directly to BpkImage where it is used on the images page.

**Comment by k0nserv** [comment]
I think the change you saw was reasonable George. At least when I tested locally it made sense.

**Pull request merged**